theme/ckeditor5-link/link.css

```diff
@@ -6,4 +6,10 @@
 /* Class added to span element surrounding currently selected link. */
 .ck-link_selected {
 	background: var(--ck-color-link-selected-background);
+	outline: 1px solid var(--ck-color-link-selected-background);
+
+	/* #155. Let's keep the same background inside Highlight feature. */
+	& mark {
```

Comment:
We shouldn't hardcode something like this. We don't even know whether highlight will produce a `<mark>` element. Is there a different solution?

Reply:
@Reinmar AFAIK this feature is strictly `<mark>`-only. I don't recall an issue/coment but the outcome was that for highlight feature we provide `<mark>` always (no way to override).

Reply:
We can try `.ck-link_selected > * ` but it can make some bugs in future (quick example: with like user markers/comments in future?).

Reply:
The problem here is that it's not going to work anyway if we'll have a font color plugin which will set inline styles. 

I need to see this live, but I think that perhaps we might just live with markers overriding link's highlight. That's still an acceptable solution to me.